### PR TITLE
INTMDB-282: Updated test and docs for alert configuration using notifications

### PR DESCRIPTION
**Pull request by coderGo93**
## Description

- Updated tests and docs to show a usage of creating an alert with two notifications.

Link to any related issue(s):
#626 
## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] Documentation fix/enhancement

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have read the Terraform contribution guidelines
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

File: mongodbatlas/resource_mongodbatlas_alert_configuration_test.go [hunk 1]
```diff
@@ -28,13 +28,15 @@ func TestAccResourceMongoDBAtlasAlertConfiguration_basic(t *testing.T) {
...
 					resource.TestCheckResourceAttrSet(resourceName, "project_id"),
+					resource.TestCheckResourceAttr(resourceName, "notification.#", "2"),
 				),
 			},
 			{
 				Config: testAccMongoDBAtlasAlertConfigurationConfig(projectID, false),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckMongoDBAtlasAlertConfigurationExists(resourceName, alert),
 					resource.TestCheckResourceAttrSet(resourceName, "project_id"),
+					resource.TestCheckResourceAttr(resourceName, "notification.#", "2"),
```
Comment (thetonymaster) [inline mongodbatlas/resource_mongodbatlas_alert_configuration_test.go:39]: Can we add specific checks on the list? Not just the number of items

**Review by thetonymaster — APPROVED** [review]
LGTM

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Review by themantissa — APPROVED** [review]
LGTM, ty!

**Pull request merged**